### ISSUE TITLE
Font Styles For Text

### ISSUE THREAD

**Issue by Rohitha-pudu** [dhruvpatidar359/nextdraw#33]
### No Font Styles

- I will work on this issue. Please assign this task to me
- Gssoc24
![image](https://github.com/dhruvpatidar359/nextdraw/assets/116823590/83a27875-5eee-4023-86ea-5f485fcf6e64)

**Comment by dhruvpatidar359** [comment]
@Rohitha-pudu sure go ahead , assigning .

**Comment by atharvarekhawar** [comment]
hey @dhruvpatidar359 , if no one is working on this issue then I am interested in solving this issue .

I am thinking of adding two more features to the text ie. font style and font weight.
eg- italic or normal , bold or lighter etc...

can you assign this issue to me.

**Comment by atharvarekhawar** [comment]
@dhruvpatidar359 , should I work on the issue?

**Comment by dhruvpatidar359** [comment]
sure , buddy

**Comment by atharvarekhawar** [comment]
hey @dhruvpatidar359 , I am getting error while committing the code due to jest , I think test folder is not created yet so it is not letting me commit , do you want me to change the husky hook to allow committing without testcases for now.

it will look something like this in package.json

"husky": {
    "hooks": {
      "pre-commit": "lint-staged && npm test --passWithNoTests",
      "pre-push": "npm test"
    }
  },

**Comment by atharvarekhawar** [comment]
> hey @dhruvpatidar359 , I am getting error while committing the code due to jest , I think test folder is not created yet so it is not letting me commit , do you want me to change the husky hook to allow committing without testcases for now.
> 
> it will look something like this in package.json
> 
> "husky": { "hooks": { "pre-commit": "lint-staged && npm test --passWithNoTests", "pre-push": "npm test" } },

@dhruvpatidar359 ?

**Comment by dhruvpatidar359** [comment]
> > hey @dhruvpatidar359 , I am getting error while committing the code due to jest , I think test folder is not created yet so it is not letting me commit , do you want me to change the husky hook to allow committing without testcases for now.
> > it will look something like this in package.json
> > "husky": { "hooks": { "pre-commit": "lint-staged && npm test --passWithNoTests", "pre-push": "npm test" } },
> 
> @dhruvpatidar359 ?

yes , do it . Mean while @VanshikaSabharwal  writes the test cases , we have to do it . 👩‍🚀
